src/_gettsim/transfers/rente.py

```diff
@@ -606,16 +606,65 @@ def ges_rente_frauen_altersgrenze(
     return out
 
 
-def _ges_rente_arbeitsl_altersgrenze(
+@dates_active(end="1996-12-31", change_name="_ges_rente_arbeitsl_altersgrenze")
+def _ges_rente_arbeitsl_altersgrenze_ohne_vertrauenss(
     birthdate_decimal: float,
-    ges_rente_params: dict,
     ges_rente_regelaltersgrenze: float,
+    ges_rente_params: dict,
 ) -> float:
     """Calculate the age, at which an unemployed is eligible to claim the full
     pension (without deductions). This pension scheme allows for early retirement
     with deductions. Hence this threshold is needed as reference for calculating the
     zugangsfaktor.
 
+    Parameters
+    ----------
+    birthdate_decimal
+        See :func:`birthdate_decimal`.
+    ges_rente_regelaltersgrenze
+        See :func:`ges_rente_regelaltersgrenze`.
+    ges_rente_params
+        See params documentation
+        :ref:`ges_rente_params <ges_rente_params>`.
+
+    Returns
+    -------
+    lowest full retirement age for unemployed.
+
+    """
+    if (
```

Comment:
This stops at the end of 1996, so I don't think any of that was law at that time?